tools/castle-editor/code/framedesign.pas

```diff
@@ -830,6 +847,44 @@ constructor TDesignFrame.Create(TheOwner: TComponent);
     Result.ShowGutter := false;
   end;
 
+  //This is a copy from FormProject - TODO: reuse the code; do not copy it
```

Comment:
Indeed I would ask to make it a single routine, used both here and in FormProject. It can be placed e.g. in `EditorUtils` unit.

Reply:
Done in cb1dd93d27cc81e2cbc992d6408480aaffda8813 , 8f8304f50948a03fb2bd46c8587cdd1583d74aba and 395797de8532d345622a9faa706040b408af99c9